spec/spec_helper.rb

```diff
@@ -11,7 +11,6 @@
     @old_env = {}
     ENV.each_key {|k| @old_env[k] = ENV[k]}
 
-    c.strict_variables = Gem::Version.new(Puppet.version) >= Gem::Version.new('3.5')
```

Comment:
Yes, we want to test with strict variables whenever possible. Using strict variables avoids many a bug.